packages/core/src/hooks/useStyle.tsx

```diff
@@ -461,6 +461,18 @@ const tlcss = css`
   .tl-grid-dot {
     fill: var(--tl-grid);
   }
+
+  .tl-user-tag {
+    color: white;
+    background: var(--tl-userColor);
+    position: relative;
+    left: 0.5em;
+    top: 1em;
+    padding: 3px 5px;
+    border-radius: 5px;
+    font-size: 10px;
+    width: fit-content;
```

Comment:
I think there is a theme you can use for those properties here

Reply:
Thanks @judicaelandria, I had a look and could not find anything suitable, did you have something specific in mind?
I replaced the `color` property with the `background` variable, and I think it makes more sense to override the `background` value with an inline style (as has been done in other components) rather than setting a global variable.
I also noticed this does not scale properly when zoomed, so I'm working on it.

Reply:
yeah if you could use stitches for styling the div
[styles](https://github.com/tldraw/tldraw/blob/main/packages/tldraw/src/styles/stitches.config.ts)
eg
const Container = styled('div', {...})

Reply:
> yeah if you could use stitches for styling the div [styles](https://github.com/tldraw/tldraw/blob/main/packages/tldraw/src/styles/stitches.config.ts) eg const Container = styled('div', {...})

You mean the `<p>` element containing the user tag? There's no `stitches.config` set up for the `core` package, should I set one up?

Reply:
Had to split the component because there was no way I could see to have both cursor and tag scale in sync while maintaining relative positions unless heavily reworking the classes. Hope that's ok.